homework6/Test/test_Task1.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from Task1 import instances_counter
+
+
+@instances_counter
+class A:
+    """some documentation"""
+
+    def get_one(self) -> int:
+        return 1
+
+
+@instances_counter
+class B:
+    pass
+
+
+@pytest.fixture
+def create_class():
+    a = A()
+    yield a
+    a.reset_instances_counter()
+
+
+@pytest.fixture
+def create_two_classes():
+    a, _, _ = A(), A(), A()
+    b = B()
+    yield a, b
+    a.reset_instances_counter()
+    b.reset_instances_counter()
+
+
+def test():
```

Comment:
The name of the test should be meaningful. Something like "test_the_name_and_docstring_preserved"